content/blog/2018-03-20-door.md

```diff
@@ -0,0 +1,179 @@
+head(responses)
+
+ORs          studies                        Odor Response
+25 ac3B Marshall.2010.WT CURLTUGMZLYLDI-UHFFFAOYSA-N       13
+29  ac4 Marshall.2010.WT CURLTUGMZLYLDI-UHFFFAOYSA-N       -1
+30  ac4      Yao.2005.WT MLFHJEHSLIIPHL-UHFFFAOYSA-N        1
+33 Or1a   Kreher.2008.EN CURLTUGMZLYLDI-UHFFFAOYSA-N        4
+34 Or1a   Kreher.2008.EN MLFHJEHSLIIPHL-UHFFFAOYSA-N       21
+36 Or2a   Kreher.2008.EN MLFHJEHSLIIPHL-UHFFFAOYSA-N       73
+
+```
+
+Please refer to the following vignettes for more tools and for a more detailed description of the above functions: [DoOR.functions](http://neuro.uni-konstanz.de/DoOR/content/doc/DoOR.functions_main.html), [DoOR tools](http://neuro.uni-konstanz.de/DoOR/content/doc/DoOR_tools.html)
+
+#### Visualizing Data
+We created various plotting functions to visualize the data in DoOR. All of the visualizations depend on `ggplot2`. One very important task of DoOR is to visualize the ensemble responses that odors elicit across the sensory neurons of *Drosophila*. `plot_al_map()` does this job by mapping the responses onto a color-coded map of the antennal-lobe. The antennal-lobe is the first odor-processing center in the insect brain. All OSNs of the same class converge onto spherical sub-structures (glomeruli) of the antennal lobe, thus the differential activity across glomeruli corresponds to the ensemble response across OSNs.
+
+Here we plot the ensemble response of 2,3-butanedione, the odor that gives margarine its buttery smell:
+
+```r
+dplot_al_map("QSJXEFYPDANLFS-UHFFFAOYSA-N", base_size = 8)
+```
+![Ensemble response of 2,3, butanedione](/img/blog-images/2018-03-20-door/bedn.png)
+
+Of course there are options to theme the output, either by passing parameters to the function or by adding `ggplot2` functions to the call:
+
+```r
+dplot_al_map("QSJXEFYPDANLFS-UHFFFAOYSA-N", tag = "", main = "",
+  legend = FALSE, scalebar = FALSE)
+```
+
+does the same as
+
+```r
+dplot_al_map("QSJXEFYPDANLFS-UHFFFAOYSA-N", tag = "", legend = FALSE) +
+  ggplot2::theme(legend.position  = "none") +
+  ggplot2::ggtitle('')
+```
+![Ensemble response of 2,3, butanedione](/img/blog-images/2018-03-20-door/bedn_blank.png)
+
+
+DoOR comes with more build-in visualizations, please see the [visualization vignette](http://neuro.uni-konstanz.de/DoOR/content/doc/DoOR_visualizations.html) for a complete list.
+
+## The DoOR Website
+The DoOR web-site is an export of the odor & OSN profiles with some background information and links to other databases. It provides a quick overview over the latest *"olfactome"* and can be found at [http://neuro.uni-konstanz.de/DoOR](http://neuro.uni-konstanz.de/DoOR).
+## Contribute!
+- You published a paper with odor responses or know a study we missed?
+  - Open an [issue](https://github.com/ropensci/DoOR.data/issues).
+- You want to build DoOR for other organisms?
+  - Just fork `DoOR.functions` and start adapting it to your needs. We are happy to assist!
+
+## Acknowledgements
```

Comment:
I'm a sucker for acknowledgements but really appreciate how thorough you are in citing people for specifics here